shell/src/main/java/tachyon/shell/command/RmrCommand.java

```diff
@@ -46,6 +47,8 @@ void runCommand(TachyonURI path) throws IOException {
       TachyonFile fd = mTfs.open(path);
       mTfs.delete(fd, options);
       System.out.println(path + " has been removed");
+    } catch (InvalidPathException e) {
+      System.out.println(path + " does not exist");
```

Comment:
I think the error message should be `is an invalid path`? I think `FileDoesNotExistException` will be thrown for the does not exist case.